var/spack/repos/builtin/packages/python/package.py

```diff
@@ -364,7 +365,9 @@ def determine_variants(cls, exes, version_str):
                 variants += "~tix"
 
         # Some modules are platform-dependent
-        if sys.platform != "win32":
+        python_version = spack.version.StandardVersion.from_string(version_str)
+        python313_version = spack.version.StandardVersion.from_string("3.13")
+        if sys.platform != "win32" and python_version < python313_version:
```

Comment:
```suggestion
        if sys.platform != "win32" and Version(version_str) < Version("3.13"):
```

Reply:
Perfect. I was looking for that. I'll also fix the imports.